src/common/templates/check-your-answers.njk

```diff
@@ -13,12 +13,44 @@
         {% endif %}
       </div>
     </dl>
+  
+    {% macro check_your_answers_row(label, value) %}
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">{{ label }}</dt>
+        <dd class="app-check-your-answers__answer">{{ value }}</dd>
+      </div>
+    {% endmacro %}
+
+    {% macro check_your_answers_row_port_code(id, label, value) %}
+      {% if value =='ZZZZ' or value =='YYYY' %}
```

Comment:
Eventually, the YYYY could be removed, but doesn't matter. Renders nothing if it is the value to represent coordinates being set.